### DISCUSSION TITLE
Reconciliation works only after page reload

### DISCUSSION THREAD

**Discussion by nottheend** [firefly-iii/firefly-iii#8117] [General]
Since one of the recent updates (currenctly on _6.0.30_), I face the following scenario:
While reconciling an account, I tick the transactions to be reconciled.
Clicking on the "Reconcile" button has no effect in the first place. Even if I change the reconciliation period and update the date via button (update), still nothing happens.
Only after I completely reload the page, the reconciliation button works.

Does someone know the reason for it or how to workaround it?

And one special thanks for implementing a functionality which **remembers** the not yet reconciled but already marked as "to be reconciled" transactions before the actual reconciliation!! 🙏🏻

**Comment by JC5** [comment]
I can't replicate this. Any errors in your browser console?

**Reply by nottheend** [reply]
Weird, this time it worked the first time. Will paste the console logs if it happens again
